_posts/2017-03-27-osd-needs-better-collaboration-tools.md

```diff
@@ -14,7 +14,7 @@ Despite the rising awareness and acceptance of UX design, in particular on the w
 
 In this post, I will argue the need for better communication and collaboration platforms by looking at some of the fundamental characteristics of open source design and development and propose some starting points.
 
-The goal of OSD is to improve the user experience of open source software. Because of the nature of open source software development, to achieve this goal, certain critical infrastructure is necessary. I am talking about a way for designers to share and build upon ideas, interact, reach consensus and collaborate with other designers and all the other stakeholders involved in the creation of software.
+The goal of OSD is to improve the user experience of open source software. Because open source software development is distributed, certain critical infrastructure is necessary. I am talking about a way for designers to share and build upon ideas, interact, reach consensus and collaborate with other designers and all the other stakeholders involved in the creation of software.
```

Comment:
@grahamperrin good stuff on kickstarting a discussion on an article! :) I think there might even be a way to embed the Discourse discussion thread on the site as some sort of comment form.